stylesheets/encode/helpers/_quote.scss

```diff
@@ -2,6 +2,11 @@
 /// @access private
 /// @param {*} $value - value to be quoted
 /// @return {String} - quoted value
+
+// [1] if $value is 'null', Sass returns 'Invalid null operation: "'"' plus null"',
```

Comment:
This comment might be causing issues with SassDoc. I get why you convert it to string before, you can safely remove it. ;)